cypress/integration/admin_account_test/report_page_layout_test.js

```diff
@@ -0,0 +1,100 @@
+            cy.contains("Save changes").click();
+            cy.get(".success").should("include.text","Successfully saved changes to report page layout.");  
+        })
+    })
+
+    it("can add description to classes", function(){
+        cy.get("@reportPageLayout").within(() => {
+            cy.get("Select").select("MRNA");
+            cy.get("select").should("have.value","MRNA");
+            cy.get("li").contains("Interactions").should("exist").click();
+            cy.get("li").contains("Exons").should("exist").click();
+            cy.get(".btn-group-sm").children().first().click();
+
+            cy.get(".classes").within(() => {
+                cy.get("textarea").clear().type("Coding regions within this gene{enter}",{delay:100});
+                cy.contains("Save").click();
+            })
+
+            cy.contains("Save changes").click();
+            cy.get(".success").should("include.text","Successfully saved changes to report page layout.");  
+        })
+    })
+
+    it("can edit the visibility of the description of a class", function(){
+        cy.get("@reportPageLayout").within(() => {
+            cy.get("Select").select("MRNA");
+            cy.get("select").should("have.value","MRNA");
+            cy.get("li").contains("Interactions").should("exist").click()
+            cy.get(".btn-group-sm").children().eq(1).click({delay:100});
+
+            cy.contains("Save changes").click();
+            cy.get(".success").should("include.text","Successfully saved changes to report page layout.");  
+        })
+    })
+
+    it("can remove a class from a category", function(){
+        cy.get("@reportPageLayout").within(() => {
+            cy.get("Select").select("MRNA");
+            cy.get("select").should("have.value","MRNA");
+            cy.get("li").contains("Interactions").should("exist").click();
+            cy.get('.css-1hwfws3').click().type("Introns{enter}",{delay:100},{force:true});
+            cy.get("ul").filter(".classes").find(".btn-fab").last().click();
+            cy.get(".btn-group-sm").eq(1).children().eq(4).click({delay:100});
+
+            cy.contains("Save changes").click();
+            cy.get(".success").should("include.text","Successfully saved changes to report page layout.");  
+        })
+    })
+
+    after(function(){
+        cy.visit("/biotestmine/report/MRNA/2000009");
+        cy.get(".report-page-heading").should("include.text","mRNA.46312");
+        cy.get('.report-table-heading').contains("Interactions")
+        .parent(".report-table")
+        .as('interactionsSection');
+
+        cy.get("@interactionsSection").within(() => {
+            cy.get(".report-item-title").should("include.text","Exons");
+            cy.get('[class="icon icon-info"]').should("exist").click();
+            cy.get(".report-item-title > .dropdown > .dropdown-menu").should("exist")
+            .find("p").should("include.text","Coding regions within this gene");
```

Comment:
Can you also assert that the Exons' `.im-table` isn't present (due to clicking the "eye" button as I explained above), then click the Exons header and assert that `.im-table` is now present?